sklearn/cluster/tests/test_hierarchical.py

```diff
@@ -870,7 +870,10 @@ def test_invalid_shape_precomputed_dist_matrix():
     # and a non square matrix is passed (PR #16257).
     rng = np.random.RandomState(0)
     X = rng.rand(5, 3)
-    with pytest.raises(ValueError, match="Distance matrix should be square, "):
+    with pytest.raises(
+        ValueError,
+        match=r"Distance matrix should be square, got matrix of shape \(5, 3\)",
```

Comment:
Tests are failing because of the `match` argument. This needs to be fixed.

Reply:
Yeap. I accepted your change but I did not change the original message.